src/poll.rs

```diff
@@ -556,7 +563,7 @@ pub struct Events {
 }
 
 /// Iterate an Events structure
-pub struct EventsIter<'a> {
+pub struct Iter<'a> {
```

Comment:
As mentioned in irc, `Iter` is actually re-exported in an `event` module, so it should be fine